.github/workflows/rust.yml

```
@@ -52,4 +51,4 @@ jobs:
     - uses: docker://ghcr.io/msrd0/cargo-doc2readme
```

```suggestion
    - uses: docker://ghcr.io/msrd0/cargo-doc2readme:nightly
```

The `--expand-macros` flag won't work on stable.